spec/views/mailchimp/nonprofit_user_subscribe.json.jbuilder_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+# License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later 
+# Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE
+require 'rails_helper'
+
+RSpec.describe '/mailchimp/nonprofit_user_subscribe.json.jbuilder', type: :view do
+  
+  describe 'adding new subscriber to nonprofit list' do 
+
+    subject(:json) do 
+      # do I need this line specifically? If so, what is it doing?
+      # where would i find these methods defined? In jbuilder docs? 
+      # view.lookup_context.prefixes = view.lookup_context.prefixes.drop(1)
+      assign(:user, create(:user, NONPROFIT_ID: '123456'))
```

Comment:
Users have roles. One of those roles, could be associated with a nonprofit. In that case, you'd need to use that. I think it'd help if we talked through how you came up with this particular attribute.